typings/artifacts.d.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+  module LH {
+    export interface Artifacts {
```

Comment:
`LH.Artifacts` is both an interface and a namespace. Typescript correctly knows if you're referring to a type (e.g. `@param {LH.Artifacts.TraceTimes} traceTimes` in a jsdoc) to look up in the namespace while if you're referring to a property on something with the `LH.Artifacts` interface (e.g. `artifacts.TraceTimes` in actual js code) to look up in the interface.

Let me know if that's confusing, but in practice they're separate worlds and it doesn't seem to cause confusion. And we get to avoid a different name for each, e.g. `LH.Artifacts.TraceTimes` vs `LH.ArtifactTypes.TraceTimes` or whatever